Exiled.CustomItems/API/Features/CustomWeapon.cs

```diff
@@ -164,8 +163,7 @@ public override Pickup Spawn(Vector3 position, Item item, Player previousOwner =
             {
                 if (!Attachments.IsEmpty())
                     firearm.AddAttachment(Attachments);
-                byte ammo = firearm.Ammo;
-                Log.Debug($"{nameof(Name)}.{nameof(Spawn)}: Spawning weapon with {ammo} ammo.", Instance.Config.Debug);
+                Log.Debug($"{nameof(Name)}.{nameof(Spawn)}: Spawning weapon with {ClipSize} ammo.", Instance.Config.Debug);
```

Comment:
ClipSize is not the same as firearm.Ammo

ClipSize is the maximum value of a fully reloaded weapon, while firearm.Ammo is the current amount of ammo loaded in the clip.

Reply:
Fair, i had an issue with `ci spawn` command, ammo magazine of all CustomWeapon that have been spawned by this command was like in default weapons, i already fixed this in 168299bf6718dada081485190b2848febb24be3f

Reply:
I just modified the wrong method